### PR TITLE
Multiple environments

### LINKED ISSUE #5: Support several environments
Useful for multiple projects
- [x] multiple rancher credentials
- [x] multiple rancher environment
- [x] multiple encryption keys

### PR DESCRIPTION
**Pull request by Dinoshauer**
Resolves #5

CC: @joshuakarjala @freeall

### CODE REVIEW

**Comment by joshuakarjala** [comment]
@freeall you check? I'm in meetings most of day :(

File: src/lib/rancher.js [hunk 1]
```diff
@@ -131,6 +119,7 @@ module.exports = function (rancherOptions, extras) {
         }
         return false;
       }).map(item => doc[item].image);
+      console.log('services', services);
```
Comment (freeall) [inline src/lib/rancher.js]: remove this log?

File: src/sealand.js [hunk 2]
```diff
@@ -38,20 +38,13 @@
 //   joshuakarjala <joshua@fluxuries.com>
 
 function hubotSealand (robot) {
+  var getEnv = require('./lib/config');
```
Comment (freeall) [inline src/sealand.js]: I think we should rename `config` module to `get-environment`

File: src/sealand.js [hunk 3]
```diff
@@ -118,45 +112,57 @@ function hubotSealand (robot) {
...
+          throw err;
+        }
+        robot.messageRoom(SLACK_ROOM_ID, `Repo: ${repoCreds.repo} Commit: ${commitHash} has been pushed to Rancher`);
+        res.send('OK');
+      });
     });
   });
 
-  robot.respond(/up (.*) (.*)/i, function (res) {
+  robot.respond(/up (.*) (.*) (.*)/i, function (res) {
```
Comment (freeall) [inline src/sealand.js]: Total Recaaaaaall!!!!!

**Comment by freeall** [comment]
Except the `config`->`get-environment` it looks good

**Pull request merged**